### PR TITLE
chore(flask): update fix release note

### PR DESCRIPTION
**Pull request by mabdinur**
## Description

Improve release note for flask fix

## Checklist
- [x] Title must conform to [conventional commit](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
- [x] Add additional sections for `feat` and `fix` pull requests.
- [x] Ensure tests are passing for affected code.
- [x] [Library documentation](https://github.com/DataDog/dd-trace-py/tree/1.x/docs) and/or [Datadog's documentation site](https://github.com/DataDog/documentation/) is updated. Link to doc PR in description.

## Reviewer Checklist
- [ ] Title is accurate.
- [ ] Description motivates each change.
- [ ] No unnecessary changes were introduced in this PR.
- [ ] PR cannot be broken up into smaller PRs.
- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
- [ ] All relevant GitHub issues are correctly linked.
- [ ] Backports are identified and tagged with Mergifyio.
- [ ] Add to milestone.

### CODE REVIEW

**Comment by mabdinur** [comment]
@Mergifyio backport 1.3

File: releasenotes/notes/fix-attribute-error-in-flask-2.2.0-94524c33b005d48a.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 fixes:
   - |
-    Flask: version 2.2.0 removes ``try_trigger_before_first_request_functions``
+    flask: remove patching for ``Flask.try_trigger_before_first_request_functions``. This method was removed in flask==2.2.0.
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/fix-attribute-error-in-flask-2.2.0-94524c33b005d48a.yaml:4]: don't think we need to mention what changed internally in flask, users don't care.

I think we can leave it as "add support for 2.2.0"

File: releasenotes/notes/fix-attribute-error-in-flask-2.2.0-94524c33b005d48a.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 fixes:
   - |
-    Flask: version 2.2.0 removes ``try_trigger_before_first_request_functions``
+    flask: add support for ``flask>=2.2``
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/fix-attribute-error-in-flask-2.2.0-94524c33b005d48a.yaml:4]:
```suggestion
    flask: add support for ``flask>=2.2.0``
```

**Review by Kyle-Verhoog — DISMISSED** [review]
nit

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by ZStriker19 — APPROVED** [review]

**Comment by mabdinur** [comment]
@Mergifyio backport 0.61

**Pull request merged**